webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const path = require('path')
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+
+const isDev = process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test'
+
+console.log(`Running webpack in ${isDev ? 'development' : 'production'} mode`)
+
+module.exports = {
+  entry: './app/frontend/index.js',
+  mode: isDev ? 'development' : 'production',
+  module: {
+    rules: [
+      {
+        test: /\.s[ac]ss$/i,
+        use: [
+          'style-loader',
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {
+              publicPath: '../'
+            }
+          },
+          'css-loader',
+          'resolve-url-loader',
+          {
+            loader: 'sass-loader',
+            options: {
+              sourceMap: isDev,
+              sassOptions: {
+                outputStyle: 'compressed'
+              }
+            }
+          }
+        ]
+      },
+      {
+        test: /\.(png|svg|jpg|gif)$/,
+        use: {
+          loader: 'file-loader',
+          options: {
+            outputPath: 'images/'
+          }
+        }
+      },
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/,
+        use: {
+          loader: 'file-loader',
+          options: {
+            outputPath: 'fonts/'
+          }
+        }
+      }
+    ]
+  },
+  output: {
+    filename: 'js/bundle.[hash].js',
+    path: path.resolve(__dirname, 'app/dist')
+  },
+  plugins: [
+    new CleanWebpackPlugin(),
+    new HtmlWebpackPlugin({
+      inject: false,
+      filename: '../views/layout.njk',
+      template: 'app/views/_layout.njk'
+    }),
+    new MiniCssExtractPlugin({
+      filename: 'css/application.[hash].css'
+    })
+  ]
+}
```

Comment:
I think this file needs a few tweaks to be similar to the map front end.  There are breaking changes in the webpack plugins installed vs what was installed in the demo web.  `[hash]` for example is now `[fullhash]`.

Also needs the names of the templates updated to match the convention used in the map front end.  Cookies needs it's own chunk added so we can minimise the cookies JS